docs-source/content/faq/namespace-management.md

```diff
@@ -47,7 +74,19 @@ suspendOnDebugStartup: false
 
 ```
 
-If you don't know the release name of the operator, you can use `helm ls` to list all the releases.
+If you don't know the release name of the operator, you can use`helm ls` to list all the releases:
+
+For Helm 2.x:
```

Comment:
If it's exactly the same for 2.x and 3.x, just say 'For Helm 2.x and 3.x:' or just 'For Helm', and use the same example for both.

Ditto for other locations.